### ISSUE TITLE
Salvage - Arrapago Remnants: 6th Floor Archaic Gear, Archaic Gears, and Archaic Rampart drop rates on cells are incorrect

### ISSUE THREAD

**Issue by DeuceFFXI** [EdenServer/community#5753]
### Checklist

- [x] I have searched for existing issues for issues like this one. The issue has not been posted.
- [] I have provided reproducible steps. (No "as the title says" posts please. ***Please try to state facts only, and avoid assumptions/opinions.***)
- [ ] I have provided my client version in the details. (type /ver into your game window)
- [x] ***I have provided proof of my claim (era videos, links to era wiki and/or forum posts)*** Lack of proof will slow down fixes and prioritization. Please provide details so that developers' research time is minimized.

### Details

At the moment Cumulus and Radiatus cells hardly drop from Archaic Gears or Archaic Ramparts respectively on the 6th Floor.  Per [this](https://www.youtube.com/watch?v=ZU0H9wOQnbg) era video from 2008, Cumulus cells are relatively common from Gears and should drop three at a time.  In addition, Archaic Ramparts should drop two cells at a time.

As can be seen in the video, these players enter the floor with a decent amount of gear, but manage to get a lot as they progress through killing the mobs to spawn the two Qiqirin.  From this perspective it also appears that enemies will have the potential to drop multiple cells of ONE type upon death.  Currently on Eden many of these enemies drop multiple cells of differing types, which appears to be partially* incorrect.

In addition, it appears that Qiqirn Treasure Hunters drop too few cells on this floor too.  The two killed in this video dropped EIGHT each.  They also dropped multiple types.  The Seasonal Treants also appear to have an incorrect drop rate as well.

In conclusion, it seems that all of the mobs on the 6th Floor of Arrapago Remnants need to have their cell drops adjusted.

**Comment by DeuceFFXI** [comment]
Another highlight from this video is the high drop rate of cells in general from mobs.  On Eden currently, it is quite common to have no cells drop at all from an enemy on the 6th floor.  Either this shouldn't happen at all, or appears to be quite rare.

**Comment by DeuceFFXI** [comment]
I've completed over a dozen captures of Arrapago Remnants on retail and logged the drop data for the relevant mobs in this bug report.

From my data the following has been proved:

**6th Floor**
- Archaic Gears drop cells in multiples of THREE; Cumulus (~30%), Undulatus (~22%)
- Archaic Gear drop cells in multiple of TWO; Duplicatus/Stratus (~26%)
- Archaic Rampart drop Radiatus cells in multiple of TWO; Radiatus (~15%)
- Seasonal Treants drop cells in multiple of TWO; Pannus/Fractus/Velum (>24%)
- Archaic Rampart on the 6th Floor _DO NOT_ drop Spissatus cells; 63 kills worth of data to prove this

In addition on the 5th Floor:
- Archaic Gears drop cells in multiples of THREE; Cumulus (~15%), Undulatus (~21%)

The discrepancy between 5th & 6th Floor droprates for Archaic Gears may be due to 30 vs 68 data points respectively.  Combining both floors drops for Gears has Cumulus ~24% and Undulatus ~22%.

![Salvage Arrapago 6th Treant Velum](https://github.com/EdenServer/community/assets/61279359/832e89ee-4bb1-425b-9db9-b56f0a27bbce)
![Salvage Arrapago 6th Rampart Radiatus](https://github.com/EdenServer/community/assets/61279359/5ea40f5a-a467-4b40-a4a1-48ad82922e7b)
![Salvage Arrapago 6th Gear Stratus](https://github.com/EdenServer/community/assets/61279359/0ef303f2-286a-4d8f-8047-7278b07d783e)
![Salvage Arrapago 6th Gears Undulatus](https://github.com/EdenServer/community/assets/61279359/7e8e5c5a-fd0f-4052-9d09-99b7ecc4b044)


[Drop data](https://docs.google.com/spreadsheets/d/1dQ3ExnPUJQzEeJ3z5iYojTkuGJ-D_CSCxrmXl1EgCc4/edit#gid=302535065)
Arrapago Captures [Playlist](https://www.youtube.com/playlist?list=PLmGJpJK88euuKM3VRzHuVXYZe00NGzZuH)

**Comment by 59blargedy** [comment]
Resolved in November 19, 2023 update.
